js/packages/teams-ai/src/prompts/Message.ts

```diff
@@ -90,17 +90,17 @@ export interface Citation {
     /**
      * The title of the citation.
      */
-    title: string;
+    title: string | null;
```

Comment:
Do we want to mark these optional?

Reply:
from what I can tell from the data coming back from azure, its either going to be a string or null 
![image](https://github.com/microsoft/teams-ai/assets/44811138/e38d672c-78b4-4138-9bad-52367a41f6e4)